cluster-autoscaler/utils/kubernetes/ready.go

```diff
@@ -48,23 +48,14 @@ func GetReadinessState(node *apiv1.Node) (isNodeReady bool, lastTransitionTime t
 			if cond.Status == apiv1.ConditionFalse || cond.Status == apiv1.ConditionUnknown {
 				canNodeBeReady = false
 			}
-			if lastTransitionTime.Before(cond.LastTransitionTime.Time) {
-				lastTransitionTime = cond.LastTransitionTime.Time
-			}
 		case apiv1.NodeOutOfDisk:
```

Comment:
In Go empty actions are ... empty actions contrary to some other languages. Please check https://github.com/golang/go/wiki/Switch. So this change is wrong.

Reply:
Yes you right, but we have less code. Not need write empty actions.
If you want I can revert this